.nvmrc

```diff
@@ -1 +1 @@
-lts/erbium
+12.18.4
```

Comment:
How come this changed?

Reply:
To ensure we use the same version as the macOS image

Reply:
There's the potential to cause confusion here, I could see somebody changing it back in future to match the other repos. Could you add a comment to say it's to match the one on CI?

Reply:
I tried but `.nvmrc` does not support comments

Reply:
A danger rule maybe?

Reply:
Ah, yeah I didn't realise we couldn't put comments in there! Danger rule would be good, could you make an enghealth card for us to do it sometime?

Reply:
Can do